README.md

```diff
@@ -1,45 +1,83 @@
+
+## Building and Installation
+
+To compile this library, you will need to download and install the [DataStax Cassandra C++ Driver](https://github.com/datastax/cpp-driver). The Datastax Cassandra C++ Driver depends on [Libuv](https://github.com/libuv/libuv) and this will need installing also.
+
+See the [INSTALL.md](https://github.com/tango-controls-hdbpp/libhdbpp-cassandra/blob/master/INSTALL.md) file for  detailed instructions on how to build and install libhdbpp-cassandra.
+
+## Running Tests
+
+The library is now partially covered with unit tests built on the Catch Unit Test framework. These are still under active development. The unit test binary is output to the bin directory.
+
+To run all the tests a running cassandra node is required to test against. See the [INSTALL.md](https://github.com/tango-controls-hdbpp/libhdbpp/blob/master/INSTALL.md) for the flags to build the unit tests. Once built, they can be run from the command line as follows. 
 
-cd libhdbpp-cassandra
-export TANGO_DIR=/usr/local/tango-9.2.5a
-export OMNIORB_DIR=/usr/local/omniorb-4.2.1
-export ZMQ_DIR=/usr/local/zeromq-4.0.7
-make
+```bash
+./bin/unit-tests
 ```
 
-By default, the Makefile assumes the Cassandra C+ driver root directory is ../../cassandra/cppDriver/cpp-driver
-You can override this default by setting CASS_CPP_DRIVER_DIR environment variable to the root directory where the Datastax driver is installed.
-E.g.:
+To look at the available tests and tags, should you wish to run a subset of the test suite (for example, you do not have a cassandra node to test against), then tests and be listed:
+
 ```bash
-export CASS_CPP_DRIVER_DIR=~/cpp-driver
+./bin/unit-tests --list-tests
 ```
 
-The datastax Cassandra C++ driver has a dependency with libuv library.
-By default, libhdbpp-cassandra Makefile assumes this libuv library can be found under ${CASS_DRIVER_DIR}/lib/libuv.
-You can override this default by setting LIBUV_ROOT_DIR environment variable to the root directory where libuv is installed.
-E.g:
+Or:
+
 ```bash
-export LIBUV_ROOT_DIR=/usr/local/libuv
+./bin/unit-tests --list-tags
+```
+
+To see more options for the unit-test command line binary:
+
+```ash
+./bin/unit-tests --help
 ```
 
 ## Compatibility
-This version was tested with Cassandra 2.2.4 and 2.2.6 and with the version 2.2.1 of the Datastax Cassandra C++ driver and with its dependency with libuv version 1.4.2.
-This version is compatible with libhdb++ library version 4.
 
-This version requires a schema having a ttl column in att_conf table.
-If you need to upgrade from a previous version of the database schema, you can add this column simply by executing the following CQL commands using cqlsh for instance:
+The library has been build and tested against certain versions of its dependencies. While it may be possible to run with newer versions, this is not supported currently since there may unknown compatibility problems. As newer versions are tested they will be added to the matrix below.
```

Comment:
there may unknown ...=> there may be unknown...?

Reply:
Yep (I was typing to fast I think)